superset/connectors/druid/models.py

```diff
@@ -292,6 +292,10 @@ def unique_name(self) -> str:
         return self.verbose_name or self.cluster_name
 
 
+sa.event.listen(DruidCluster, "after_insert", security_manager.set_perm)
+sa.event.listen(DruidCluster, "after_update", security_manager.set_perm)
```

Comment:
Is it possible to implement this feature without model triggers? I find that they tend to embed business logic in the ORM and can lead to difficult-to-diagnose bugs down the line.

Reply:
@willbarrett it would be a large refactor of existing functionality and all permissions model was based on using model triggers. It is definitely possible, however we should get buy-in from project maintainers & probably do it outside of this PR.

Reply:
OK then. I'll make a task for myself to open up a SIP. Thanks!